addons/website_form/static/src/snippets/s_website_form/options.js

```diff
@@ -821,6 +830,22 @@ options.registry.WebsiteFieldEditor = FieldEditor.extend({
         // We need to reload the existing type list.
         this.rerender = true;
     },
+    /**
+     * Generates new ids for the cloned elements.
+     *
+     * @override
+     */
+    onClone() {
+        const clonedEl = this.$target[0];
+        const newId = this._generateUniqueID();
+        for (const el of clonedEl.querySelectorAll('[id]:not(div)')) {
+            el.id = newId;
+        }
+        for (const el of clonedEl.querySelectorAll('[for]')) {
+            el.setAttribute('for', newId);
+        }
```

Comment:
I don't understand the loop here.
This is triggered when you clone a field/form, but is actually related to a single field (not the form). How could there be more than one element here?
And even if there were more than one element, the generateId should be inside the loop anyway?
Maybe I am missing something

Reply:
Are you talking about multiple checkbox and stuff?

It doesn't seem to work, see, when creating one, you get a different ID by checkbox, incremented by one:

![image](https://user-images.githubusercontent.com/30048408/128740401-8a62bd29-726a-4148-8f80-10925623fba6.png)

But when you clone it, both get the same ID:

![image](https://user-images.githubusercontent.com/30048408/128740447-892fffb5-6102-481f-80d0-823fba7c8111.png)

Reply:
Indeed, then every label of the checkbox will target the first checkbox, so clickiing on label from checkbox 3 will check checkbox 1

Side note: When submitting a form with multiple checkboxes, the selected checkboxes are not send :grimacing:  ? (not related to your branch)

Reply:
I pushed a new version of the commit using _replaceField, which rerenders the field template with the same parameters of the clone and a new ID. 
That  way, for multiple inputs, the ids are created incrementally as defined in the template, based on the newly generated ID, and we don't have to duplicate this logic in the onClone method.

Reply:
Minor remark but shouldn't we add a note in the docstring of the onclone to mention the special case of ID of checkbox and radio (same ID but suffix incremented by 1)?
I don't think it is obvious, and one might change the code one day not noticing that particular behavior.